x/marker/spec/12_transfers.md

```diff
@@ -0,0 +1,264 @@
+# Transfers
+
+There are some complex interactions involved with transfers of restricted coins.
+
+<!-- TOC -->
+  - [General](#general)
+  - [Definitions](#definitions)
+    - [Transfer Permission](#transfer-permission)
+    - [Required Attributes](#required-attributes)
+    - [Individuality](#individuality)
+    - [Deposits](#deposits)
+    - [Bypass Accounts](#bypass-accounts)
+  - [Send Restrictions](#send-restrictions)
+    - [Flowcharts](#flowcharts)
+    - [Quarantine Complexities](#quarantine-complexities)
+
+## General
+
+Accounting of restricted coins is handled by the bank module. Restricted funds can be moved using the bank module's `MsgSend` or `MsgMutliSend`. They can also be moved using the marker module's `MsgTransferRequest`.
+
+During such transfers several things are checked using a `SendRestrictionFn` injected into the bank module. This restriction is applied in almost all instances when funds are being moved between accounts. The exceptions are delegations, undelegations, minting, burning, and marker withdrawals.
+
+<!-- TODO: Add notes about IBC movement too -->
+
+## Definitions
+
+### Transfer Permission
+
+One permission that can be granted to an address is `transfer`. An address with `transfer` permission can utilize `MsgTransferRequest` to move restricted funds from one account to another. If the marker allows forced transfer, the source account can be any account, otherwise, it must be the admin's own account.
```

Comment:
```suggestion
One permission that can be granted to an address is `transfer`.  The `transfer` permission is granted to accounts that represent a "Transfer Agent" or "Transfer Authority" for restricted marker tokens. An address with `transfer` permission can utilize `MsgTransferRequest` to move restricted funds from one account to another. This authority can be granted (see `authz`) to another address allowing them to use the `MsgTransferRequest` endpoint.

An additional transfer capability is the "Forced Transfer" permission.  This permission changes the nature of the token itself such that accounts no longer fully self-custody the marker token. The Transfer Agent on a "Forced Tranfer" marker has the ability to move the funds from one account to another unilaterally.  Note that the source account must be a "standard signing account".  Smart contracts or other module and protocol controlled accounts are not valid sources to force transfers from.
```

Reply:
I left off the
> This authority can be granted (see `authz`) to another address allowing them to use the `MsgTransferRequest` endpoint.

It's not really the authority that's being granted with `authz`, just the ability to use a `MsgTransferRequest`. That's just normal `authz` operation, though, so I'm not sure it needs to be called out. You can't use `authz`, though, to convey transfer permission in a way that `MsgSend` could be used. There could also be some confusion with the `MarkerTransferAuthorization` which is issued by someone with some restricted coins, but without transfer permission to someone with transfer permission. But that's discussed in `11_authorization.md`.